Lib/fontTools/misc/bezierTools.py

```diff
@@ -437,10 +571,19 @@ def _splitCubicAtT(a, b, c, d, *ts):
 
 def solveQuadratic(a, b, c,
         sqrt=sqrt):
-    """Solve a quadratic equation where a, b and c are real.
-        a*x*x + b*x + c = 0
-    This function returns a list of roots. Note that the returned list
-    is neither guaranteed to be sorted nor to contain unique values!
+    """Solve a quadratic equation.
+
+    Solves *a*x*x + b*x + c = 0* where a, b and c are real.
+
+    Args:
+        a: coefficient of *x²*
+        b: coefficient of *x*
+        c: constant term
+        sqrt: square root function to be used
```

Comment:
I believe the fact `sqrt` is among the parameters is just an optimization (though it is doubtful it actually makes a difference in real world).
There used to be a recommendation about that in itertools documentation for python 2.7
https://docs.python.org/2/library/itertools.html

> many of the above recipes can be optimized by replacing global lookups with local variables defined as default values

But it seems that comment has been removed in more recent versions of the python documentation. Maybe it no longer applies. In any case, you don't need to mention those here.
I noticed a similar thing with `min` and `max` functions elsewhere in fonttools (I don't exactly recall where).
If you like you could rename the parameter `_sqrt=sqrt` to make it clear that is somewhat private.
Or you can profile and decide to remove it. Sure, not in this same PR.

Reply:
Yeah, in the old days this really made a performance difference (global lookups are much slower than locals), and this arg should be scrapped.